.shippable.yml

```diff
@@ -0,0 +1,18 @@
+build_image: freedomjs/freedom
+language: node_js
+node_js:
+- '0.10'
+notifications:
+  email:
+    recipients:
+      - freedom@cs.washington.edu
+    on_success: change
+    on_failure: always
+before_install:
+  - "export DISPLAY=:10.0"
+  - "Xvfb :10 -screen 0 1280x1024x24 &"
+install:
+  - "git pull"
```

Comment:
I guess, although `apt-get` of a specific chrome will allow similar control on the shippable image. I don't feel super-strongly either way, but would prefer to have shippable doing more of the work :ok_hand: